client/src/main/com/sinch/sdk/domains/numbers/api/v1/adapters/NumbersService.java

```diff
@@ -98,6 +108,50 @@ public WebHooksService webhooks() {
     return this.webhooks;
   }
 
+  public AvailableNumberListResponse searchForAvailableNumbers(
+      AvailableNumberListRequest parameters) throws ApiException {
+    return available().list(parameters);
+  }
+
+  /**
+   * Get available number information by phone number
+   *
+   * <p>This endpoint allows you to enter a specific phone number to check if it's available for use
+   *
+   * @param phoneNumber Phone number
+   * @return Available number information
+   * @since 1.2
+   */
+  public AvailableNumber checkAvailability(String phoneNumber) throws ApiException {
+    return available().checkAvailability(phoneNumber);
+  }
+
+  public ActiveNumber rent(String phoneNumber, AvailableNumberRentRequest parameters)
```

Comment:
Only the `checkAvailability` method gets to have a javadoc

Reply:
Removed javadoc here: we are in "adapters" it do not have to be documented (and is not)